app/integrations/opsgenie.py

```diff
@@ -17,8 +18,35 @@ def get_on_call_users(schedule):
         return []
 
 
+# Create an Opsgenie alert. This is used to notify the on-call users
+def create_alert(description):
+    content = api_post_request(
+        "https://api.opsgenie.com/v2/alerts",
+        {"name": "GenieKey", "token": OPSGENIE_KEY},
+        {
+            "message": "Notify API Key has been compromised!",
+            "description": f"{description}",
+        },
+    )
+    try:
+        data = json.loads(content)
+        return data["result"]
+    except Exception:
+        return "Could not issue alert to Opsgenie!"
```

Comment:
Minor tweak - could you log the exception here along with the `content` var here just to help with future troubleshooting.

Reply:
Thanks for the suggestion Pat and it is done!